.travis.yml

```
@@ -57,7 +57,7 @@ before_install:
       sudo wget -O - http://packages.couchbase.com/ubuntu/couchbase.key | sudo apt-key add -
       echo "deb http://packages.couchbase.com/ubuntu xenial xenial/main" | sudo tee /etc/apt/sources.list.d/couchbase.list
       sudo apt update
-      sudo apt install -y librabbitmq-dev libsodium-dev libcouchbase-dev zlib1g-dev
+      sudo apt install -y librabbitmq-dev libsodium-dev libcouchbase-dev zlib1g-dev php-uuid
```

```suggestion
      sudo apt install -y librabbitmq-dev libsodium-dev libcouchbase-dev php-uuid zlib1g-dev
```

---

Since this will conflict anyway, perhaps sort all of them?

```suggestion
      sudo apt install -y libcouchbase-dev librabbitmq-dev libsodium-dev php-uuid zlib1g-dev 
```